packages/koa-shopify-auth/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
 
+- Include `prefix` when redirect to the root endpoint
```

Comment:
Last nit: please add a link to the PR at the end of this like (eg. `[[#PR_NUMBER]](LINK)`)

Reply:
Cheers, done